### ISSUE TITLE
Preserve linebreaks while copying

### ISSUE THREAD

**Issue by DorKeinath** [PrivateBin/PrivateBin#430]
## Steps to reproduce

1. Create a discussion.
2. Write an answer with several lines separated through line breaks.
2. Mark this lines, copy it and paste it in an editor.

### What happens

The text is inserted without line breaks.

### What should happen

The format should be preserved.

**Browser: Firefox 66**

**PrivateBin version: 1.2.1 and 1.1**

I can reproduce this issue on <https://privatebin.net>: Yes

**Comment by rugk** [comment]
Indeed, the issue seems to be that we do not use real `<br>` line breaks, but just the Unicode characters for new line…(?)

![image](https://user-images.githubusercontent.com/11966684/57633532-088c2780-75a4-11e9-91e4-4fc18d2da471.png)

**Comment by Ribas160** [comment]
@rugk 
Is it task still relevant?
I checked locally and seems that text pasted in an editor displays correctly - with line breaks.
May be it was a problem for older versions of browsers. (Checked in Chrome 131 and Firefox 134)

**Comment by elrido** [comment]
I can't reproduce it either - lets close it for now. Anyone feel free to re-open, if it is still occurring. Please do mention the browser and share the example comment, if possible, so we can recreate it.
